src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css

```
@@ -0,0 +1,109 @@
+.bleedWrapper {
+  background-color: var(--color-black);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: 465px;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: flex;
+    flex-direction: column;
```

Can this just be replaced with `display: block`? A column flex is unnecessary unless if you are actually trying to get items to grow proportionally with `flex-grow` or `flex-shrink`.

```suggestion
    display: block;
```

---

ok